custom/icds_reports/migrations/0151_drop_duplicate_indexes.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.26 on 2019-11-25 21:31
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from corehq.sql_db.operations import RawSQLMigration
+
+migrator = RawSQLMigration(('custom', 'icds_reports', 'migrations', 'sql_templates'))
+
+
+class Migration(migrations.Migration):
+    atomic = False
```

Comment:
build is failing with this error:
```
django.db.utils.InternalError: DROP INDEX CONCURRENTLY cannot run inside a transaction block
```

Not sure why since you've marked this as non-atomic.

Reply:
I spent the past hour or so digging in django internals/our migrations trying to figure this one out and could not figure this one out, but did see that we never use the atomic=False pattern with our custom migrator. i may just make this run with django's RunSQL pattern unless another idea comes to me

Reply:
odd, I don't see how using the migrator would have an impact. It just returns a version of `RunSQL`

Reply:
I'm not sure what's going on,but it does look like we never try to drop index concurrently in a transaction. I wonder if a lot of our reverse sql commands are actually broken and would fail in a similar way?

I took this out of a non-atomic migration and just dropped if it exists. Most of hte se tables are somewhat old so it should drop quickly, but I could run thsi outside of a deploy as well. Also looks like this PR is now failing on some other tests that other tests on master are failing on